spec/main_spec.rb

```diff
@@ -0,0 +1,34 @@
+require './lib/logic'
+
```

Comment:
- Using 'require_relative' for files in your directory instead of 'require' is a ruby convention.